paloalto/rules/test_rule.yaml

```diff
@@ -0,0 +1,15 @@
+---
+    name: "test_rule"
+    pack: "paloalto"
+    description: "Test rule dumping http webhook payload to a file"
+    enabled: false
+
+    trigger:
+        type: "core.st2.webhook"
+        parameters:
+            url: "paloalto"
+
+    action:
+        ref: "core.local"
+        parameters:
+            cmd: "echo \"{{trigger.body}}\" >> ~/st2.paloalto.out"
```

Comment:
This looks like just a simple test rule and not pack specific. Since this is a public pack, can you remove this test rule?

Reply:
I can but test_rule was meant to be used for testing , sending test hook from paloalto firewall without updating firewall